frontend/src/pages/Player/PlayerHook/PlayerState.ts

```diff
@@ -555,6 +555,9 @@ export const PlayerReducer = (
 		case PlayerActionType.setIsLiveMode:
 			s.isLiveMode = handleSetStateAction(s.isLiveMode, action.isLiveMode)
 			s = initReplayer(s, events, !!s.replayer?.config.mouseTail)
+			analytics.track('Session live mode toggled', {
```

Comment:
just checking for my own understanding, this should get triggered from https://github.com/highlight/highlight/blob/18f8b27ff9094cb1feed0d9c48c5500f8b46cda7/frontend/src/pages/Player/Toolbar/ToolbarControlBar/ToolbarControlBar.tsx#L219-L232

so this should cover all the live mode cases

Reply:
Yeah, exactly!